blog/_posts/2021-12-20-discord.md

```diff
@@ -0,0 +1,176 @@
+\#scala-contributors, \#announcements, \#jobs, \#tooling, \#scala-js,
+\#scala-native, and others.  New channels can be proposed on \#admin.
+
+## What about Gitter?
+
+Adopting Discord means we have moved away from Gitter.  Most of our
+old Gitter rooms are now closed.  The main room remains open, but
+mainly so we can help people find us on Discord.
+
+Gitter served the Scala community extremely well for more than six
+years.  We applaud the Gitter team for their pioneering efforts, which
+revolutionized public online chat for programmers.  We also thank
+Gitter for providing us with free, high-quality service for so long.
+
+## How is Discord different?
+
+A Discord "server" may contain many "channels", also called rooms.
+
+When you join a Discord server, all of the public channels on that
+server are immediately visible to you. Unlike Gitter, you don't have
+to join each channel individually.  (But you can individually mute
+channels that don't interest you.)
+
+There is a desktop client and a mobile client (for iOS and Android).
+Both are high quality.
+
+Former Gitter users will notice a number of quality-of-life
+and user-interface improvements such as reactions and replies.
+
+## What other chat servers exist?
+
+Scala enthusiasts might also be interested in joining the following
+Discord servers:
+
+* **[Scalameta](https://discord.gg/RFpSVth)**: Scalameta-based tooling: Metals, Scalameta, Scalafix, Scalafmt, and Mdoc
+* **[Typelevel](https://discord.gg/XF3CXcMzqD)**: the Typelevel ecosystem for pure-functional programming in Scala
+* **[ZIO](https://discord.gg/2ccFBr4)**: Type-safe, composable asynchronous and concurrent programming for Scala
+* **[Functional Programming](https://discord.gg/K6XHBSh)**: not Scala-specific, but includes a Scala channel
+
+And although we have closed down all but one of the official Scala
+Gitter rooms, as of December 2021 there are still some active Gitter
+rooms out there devoted to particular libraries or topics, for example
+[playframework/playframework](https://gitter.im/playframework/playframework).
+
+## What about non-English languages?
+
+Our website's [community
+page](https://www.scala-lang.org/community/#chat-rooms) lists chat
+rooms and servers for many other languages besides English. Discord,
+Gitter, and Telegram are all represented.
+
+As of December 2021, the list is probably not very up-to-date.  We
+invite the community to submit pull requests that will make it more
+current, one language at a time.  The file to edit on GitHub is
+[community/index.md](https://github.com/scala/scala-lang/blob/main/community/index.md).
+
+## How did we decide to switch to Discord?
+
+The community discussion of record on this began in October 2019
+on the following Discourse thread:
+
+* https://users.scala-lang.org/t/can-we-have-an-official-scala-discord-server/5157
+
+Among the objections raised were:
+
+* Discord is a proprietary, closed-source platform.
```

Comment:
perhaps it might be helpful to mention that people who prefer to avoid proprietary platforms may `#scala-lang:matrix.org` on matrix, as long as they realize the bridging to the corresponding Discord room is not perfect?

Reply:
I have no firsthand experience with this, but I have seen people using it so I guess it more or less works.

I have added a commit to this PR that adds information about Matrix to both the blog post, and to the chat section of the Community page. Can you let me know if I got it right in both places...?

Reply:
Looks good AFAICS, thanks!